cmd/substreams/graph.go

```diff
@@ -2,16 +2,22 @@ package main
 
 import (
 	"fmt"
+	"github.com/streamingfast/cli"
+	"github.com/streamingfast/substreams/tools"
 
 	"github.com/spf13/cobra"
 	"github.com/streamingfast/substreams/manifest"
 )
 
 var graphCmd = &cobra.Command{
-	Use:          "graph <manifest_file>",
-	Short:        "GenerateProto mermaid-js graph document",
-	RunE:         runManifestGraph,
-	Args:         cobra.ExactArgs(1),
+	Use:   "graph [<manifest_file>]",
+	Short: "GenerateProto mermaid-js graph document",
+	RunE:  runManifestGraph,
+	Long: cli.Dedent(`
+		GenerateProto mermaid-js graph document. The manifest is optional as it will try to find one in your pwd if nothing entered.
+		You may enter a dir that contains a 'substreams.yaml' file in place of <manifest_file>
```

Comment:
We should probably have the same description for the "optional" description for each command. The current `codegen` seems better, I would just merge there the two sentences in a single "paragraph".

```
The manifest is optional as it will try to find one a file named 'substreams.yaml' in current working directory if nothing entered. You may enter a directory that contains a 'substreams.yaml' file in place of '<manifest_file>'.
```